cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -34,12 +34,16 @@
   @Override
   public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
     if (t instanceof BadRequestException) {
+      logWarnings(request, t);
```

Comment:
What I mean is don't log it as warning. If really need to log, it's more like a trace.